metric/common.go

```diff
@@ -54,9 +54,26 @@ type baseEntry interface {
 	read(t time.Time) metricdata.Point
 }
 
+func (bm *baseMetric) startTime() time.Time {
+	switch bm.bmType {
+	case cumulativeInt64:
```

Comment:
Nit: all four cases can be combined to one line `case cumulativeInt64, cumulativeFloat64...`

Reply:
thx. I was looking for that syntax.